connectors/src/connectors/google_drive/index.ts

```diff
@@ -309,6 +317,7 @@ export async function retrieveGoogleDriveConnectorPermissions(
     return new Err(new Error("Connector not found"));
   }
   const authCredentials = await getAuthObject(c.connectionId);
+  const memoKey = randomUUID();
```

Comment:
So we memoize per request but don't memoize across request. Should use the current time modulo something like 15mn or 30mn so that we get hits during a full interaction with the UI?

Reply:
>  so that we get hits during a full interaction with the UI?

Should we use redis then ? I don't think we have anything that guarantees that subsequent calls from `front` to `connectors` get routed to the same k8s pod, so at best this would be "sometimes hits"